Requesting a pull to mozilla:master from k88hudson:t2522

k88hudson opened the pull request.

Commit:
[t2522] Changes to landing page

mjschranz commented on public/index.html:
```diff
@@ -209,19 +225,45 @@ <h5 class="title">
+
+    (function() {
+      var IFRAME_WIDTH = 560,
+          IFRAME_HEIGHT = 340,
```
These don't appear to be used for anything.

mjschranz commented on public/index.html:
```diff
@@ -26,11 +26,13 @@
                   <li class="home"><a href="/">Home</a></li>
                 </ul>
               </nav>
+              -->
```
If you don't need these elements, might as well remove them.

mjschranz replied:
You missed this one, unless there is a reason you are keeping this one here.

mjschranz commented on public/index.html:
```diff
@@ -209,19 +222,47 @@ <h5 class="title">
+        modal.addEventListener( "click", function closeModal() {
+          this.style.opacity = "";
+          this.style.visibility = "";
+          console.log( modalInner );
```
Rogue console.log

mjschranz commented on public/index.html:
```diff
@@ -67,99 +67,112 @@
+              <div class="span5">
+                <h4><a href="/templates/basic/?savedDataUrl=tour.json">Lego</a></h4>
+                  <div class="thumbnail preview-thumbnail">
+                    <img src="http://dl.dropbox.com/u/29163874/Screenshots/7m.png" alt="http://dev.butyr.org/v/gh_.html">
```
Technically I suppose there isn't anything wrong with using alt this way but as far as I know it isn't intended.

Why not just use your own attribute on these like, data-iframe-url or something instead? That way it's more obvious what the intent is.

secretrobotron replied:
I agree with Matt.

secretrobotron committed:
[t2496]

1. webmakernav require entry-point
2. preventative measures for feedback button
3. margin reset for webmakernav .primary

Commit:
[t2522] New logo

secretrobotron commented on public/index.html:
```diff
@@ -15,151 +15,140 @@
-                  <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.mp4" type="video/mp4">
-              </video>
+            <div class="span5">
+              <iframe class="iframe-template" src='http://dev.butyr.org/v/gh_.html' width='560' height='340' mozallowfullscreen webkitallowfullscreen allowfullscreen></iframe>
```
Mixing quotes.

secretrobotron commented on public/index.html:
```diff
@@ -15,151 +15,140 @@
+                <h4><a href="/templates/basic/?savedDataUrl=tour.json">Neighbourhood Tour</a></h4>
+              </div>
+              <div class="span7">
+                <iframe class="iframe-template" src='http://dev.butyr.org/v/gh_.html' width='560' height='340' mozallowfullscreen webkitallowfullscreen allowfullscreen></iframe>
```
Mixing quotes.

secretrobotron commented on public/index.html:
```diff
@@ -209,19 +198,47 @@ <h5 class="title">
+          previews = document.querySelectorAll( ".preview-thumbnail" ),
+          i, l;
+
+      function spawniFrame( e ) {
```
Fix your camelCase in `spawniFrame`.

secretrobotron commented on public/index.html:
```diff
@@ -209,19 +198,47 @@ <h5 class="title">
+
+      function spawniFrame( e ) {
+        var newiFrame = iframeTemplate.cloneNode( true );
+        newiFrame.src = this.querySelector( "img" ).getAttribute( "alt" );
```
See https://github.com/mozilla/butter/pull/1147/files#L2R93

secretrobotron commented on public/media/css/page.css:
```diff
@@ -268,13 +321,71 @@ Home styles
+}
+
+#bottom-section img {
+  display: none; /* for now */
```
Does this need to be here? I fear this and the actual images in the html will be left in without further consideration.

secretrobotron commented on public/index.html:
```diff
@@ -16,150 +16,128 @@
-                  </ul>
+          <div id="projects">
+            <div class="row">
+                <div class="span10">
```
Indentation
